src/Telemetry/Microsoft.DotNet.AzureDevOpsTimelineTests/AzureDevOpsTimelineTests.cs

```diff
@@ -0,0 +1,308 @@
+using FluentAssertions;
+using Microsoft.DotNet.Internal.AzureDevOps;
+using Microsoft.DotNet.Internal.Testing.Utility;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Internal;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.AzureDevOpsTimeline.Tests
+{
+    public class AzureDevOpsTimelineTests
+    {
+        private class TestData : IDisposable
+        {
+            public readonly AzureDevOpsTimeline Controller;
+            private readonly ServiceProvider _services;
+
+            public TestData(AzureDevOpsTimeline controller, ServiceProvider services)
+            {
+                Controller = controller;
+                _services = services;
+            }
+
+            public void Dispose()
+            {
+                _services?.Dispose();
+            }
+        }
+
+        private static TestData SetUp(
+            ITimelineTelemetryRepository timelineTelemetryRepository, 
+            IAzureDevOpsClient azureDevOpsClient,
+            ISystemClock systemClock)
+        {
+            ServiceCollection collection = new ServiceCollection();
+            collection.AddOptions();
+            collection.AddLogging(l =>
+            {
+                l.AddProvider(new NUnitLogger());
+            });
+
+            collection.Configure<AzureDevOpsTimeline>(options =>
```

Comment:
Does this call actually do anything?  Also, the "AzureDevOpsTimeline" isn't an "Options", so this configure is a little weird.

Reply:
It doesn't. I ended up being able to avoid needing to mock config values. Will remove.